src/Component/Login.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { GoogleLogin } from 'react-google-login';
+import { useNavigate } from 'react-router-dom';
+// require('dotenv').config();
```

Comment:
Can we remove this code if not being used?

Reply:
I remove all comment and not used code in new PR!